scripts/pr-review.py

```
@@ -202,13 +202,11 @@ def summarize_usage(data: dict) -> str:
 def extract_chat_content(data: dict) -> str:
     """Extract visible text from a chat-completions response."""
     choices = data.get("choices", [])
-    if not choices:
-        raise LLMReviewError(
-            "LLM returned no choices. Raw response: " + json.dumps(data)[:500]
-        )
+    if not isinstance(choices, list) or not choices:
+        raise LLMReviewError("LLM returned no choices.")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)